### ISSUE TITLE
VAC BANS

### ISSUE THREAD

**Issue by HRVAT007** [Jire/Abendigo#116]
I got VAC banned for using abendigo. I think it got detected.

**Comment by leandrosantos23** [comment]
yes, me too!

**Comment by benjaminseng990** [comment]
Do you guys have any solid evidence that you got vac ban like a picture of your steam profile getting vac ban

**Comment by JustinPandas** [comment]
You can check my profile.
http://steamcommunity.com/id/AdoptedPandas/

**Comment by leandrosantos23** [comment]
http://steamcommunity.com/id/123ddfd/

**Comment by leandrosantos23** [comment]
All my friends banned

**Comment by benjaminseng990** [comment]
I just played 1 competitive match and I had been vac banned so far, so it should be safe and undetected for now. Did you use any external hacks other than abendigo

**Comment by jo77528** [comment]
It's detected for sure LOL I just got VAC banned too, and i only use abendigo

**Comment by jo77528** [comment]
Once VAC scans ur comp ur gonna get banned

**Comment by MaGoKz** [comment]
GG VAC BAN

**Comment by taariqamozilla** [comment]
Same I got a vac (on my main) and im really pissed (but I guess its my fault) still pissed tho

**Comment by txpxsx** [comment]
Do we need to format pc after VAC ban?

**Comment by HRVAT007** [comment]
I used only abendigo too and i got VAC not overwatch but VAC. I know the difference between those 2 bans.

**Comment by txpxsx** [comment]
Should we format the pc? Since valve scans it...I'm afraid my other accounts will be affected

**Comment by benjaminseng990** [comment]
I don't think you need to format it to be safe just avoid playing on valve servers for now and wait for Jire to fix the problem first

**Comment by JuiceShack** [comment]
Have not played in a few days. But got VACed today. 
http://steamcommunity.com/profiles/76561198336726683/

**Comment by benjaminseng990** [comment]
Have this issue gotten worse or is it getting better?

**Comment by davewolax** [comment]
I have been using Abendigo for more than 6 month now. and I didn't get a ban. I'm still using it and its totally safe. you guys probably messed up the configurations otherwise this is legit 100% not detected.

**Comment by MaGoKz** [comment]
@davewolax  Hey Bruh what is ur Config?...

**Comment by MaGoKz** [comment]
@davewolax i just turn on the RCS and the FOVAim... -_-

**Comment by davewolax** [comment]
@MaGoKz Only GlowESP and FOVAim and make sure you build it once a while

**Comment by davewolax** [comment]
No ban yet. clean and alright.

**Comment by davewolax** [comment]
I got banned. this is detected and you should not be using it. Add Jire on skype (jire..) keep spamming him for getting your account banned and fuck him up. ask him to pay for your CS GO

**Comment by smurfmannen** [comment]
@davewolax If this FREE cheat has been ud for six months, you should show some more respect towards Jire. He has made an amazing job with this cheat keeping it ud for six months. And if you are mad because you got banned you should keep your feelings for yourself and not tell us to "fuck him up"

**Comment by Jire** [comment]
@davewolax Yeah, keep calling non-stop demanding help when you haven't paid a dime.. One day you'll learn maybe.

**Comment by MaGoKz** [comment]
@Jire Where is the Cheat Life?... Can't wait...

**Comment by jo77528** [comment]
@Jire isn't our slave, he did us all a huge favour by allowing free, secure, and top notch cheats for six months. You didn't do anything to deserve it, so just stfu and leave Jire alone.
